.github/workflows/bcny-ci.yml

```diff
@@ -58,3 +58,38 @@ jobs:
         with:
           name: sentry-native-windows-${{ matrix.arch }}
           path: sentry-native-development
+
+      - name: Package Sentry
+        run: |
+          $suffix = "${{ matrix.arch }}".ToLower()
+          @"
+          <?xml version="1.0" encoding="utf-8"?>
+          <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+            <metadata>
+              <id>io.sentry.sentry-native.win.$suffix</id>
+              <version>0.0.0.0</version>
+              <title>Sentry Native SDK</title>
+              <description>C++ Sentry Native SDK</description>
+              <authors>Functional Software, Inc.</authors>
+              <projectUrl>https://github.com/getsentry/sentry-native</projectUrl>
+              <repository type="git" url="https://github.com/getsentry/sentry-native" branch="main" />
+            </metadata>
+            <files>
+              <file src="`$DESTDIR`$\bin\crashpad_handler.exe" target="bin" />
+              <file src="`$DESTDIR`$\bin\crashpad_handler.pdb" target="bin" />
+              <file src="`$DESTDIR`$\bin\crashpad_wer.dll" target="bin" />
+              <file src="`$DESTDIR`$\bin\crashpad_wer.pdb" target="bin" />
+              <file src="`$DESTDIR`$\bin\sentry.dll" target="bin" />
+              <file src="`$DESTDIR`$\bin\sentry.pdb" target="bin" />
+              <file src="`$DESTDIR`$\lib\sentry.lib" target="lib" />
+              <file src="`$DESTDIR`$\include\sentry.h" target="include" />
```

Comment:
How will it work when we package the application for ARM and x86? Will we have to copy two different `crashpad_handler.exe` in? Do we have to rename them? Will there be a clash?

Reply:
The package would be different (io.sentry.crashpad.arm64 vs io.sentry.crashpad.amd64)